apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { FunctionComponent, ReactNode, useCallback } from 'react';
+import { Tabs } from '@/common/components/organisms/Tabs/Tabs';
+import { TabsList } from '@/common/components/organisms/Tabs/Tabs.List';
+import { TabsTrigger } from '@/common/components/organisms/Tabs/Tabs.Trigger';
+import { TabsContent } from '@/common/components/organisms/Tabs/Tabs.Content';
+import { Link, useNavigate } from 'react-router-dom';
+import { ChevronLeft } from 'lucide-react';
+import { Button } from '@/common/components/atoms/Button/Button';
+import { useZodSearchParams } from '@/common/hooks/useZodSearchParams/useZodSearchParams';
+import { z } from 'zod';
+import { Badge } from '@ballerine/ui';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { BusinessReportStatus } from '@/domains/business-reports/fetchers';
+import dayjs from 'dayjs';
+
+export const MerchantMonitoringBusinessReport: FunctionComponent = () => {
+  const tabs = [
+    {
+      label: 'Summary',
+      value: 'summary',
+      content: 'Summary content',
+    },
+    {
+      label: "Website's Company",
+      value: 'websitesCompany',
+      content: 'Website content',
+    },
+    {
+      label: 'Website Line of Business',
+      value: 'lineOfBusiness',
+      content: 'Line of Business content',
+    },
+    {
+      label: 'Website Credibility',
+      value: 'credibility',
+      content: 'Credibility content',
+    },
+    {
+      label: 'Ecosystem and Transactions',
+      value: 'ecosystemAndTransactions',
+      content: 'Ecosystem and Transactions content',
+    },
+    {
+      label: 'Ads and Social Media',
+      value: 'adsAndSocialMedia',
+      content: 'Ads and Social Media content',
+    },
+  ] as const satisfies ReadonlyArray<{
+    label: string;
+    value: string;
+    content: ReactNode | ReactNode[];
+  }>;
+  const tabsValues = tabs.map(tab => tab.value);
+  const MerchantMonitoringBusinessReportSearchSchema = z.object({
+    activeTab: z
+      .enum(
+        // @ts-expect-error - zod doesn't like we are using `Array.prototype.map`
+        tabsValues,
+      )
+      .catch(tabsValues[0]!),
+  });
+  const [{ activeTab }] = useZodSearchParams(MerchantMonitoringBusinessReportSearchSchema);
+  const navigate = useNavigate();
+  const onNavigateBack = useCallback(() => {
+    const previousPath = sessionStorage.getItem(
+      'merchant-monitoring:business-report:previous-path',
+    );
+
+    if (!previousPath) {
+      navigate('../');
```

Comment:
validate when a user get into this page by link.

Reply:
This should be handled only for drawers, dialogs, modals, sheets, and so on. In this case its a go back button.